mlflow/tracing/fluent.py

```diff
@@ -293,6 +313,7 @@ def search_traces(
         with mlflow.start_span(name="non_dict_span") as span:
             span.set_inputs(["a", "b"])
             span.set_outputs([1, 2, 3])
+
         mlflow.search_traces(
```

Comment:
This example should work once https://github.com/mlflow/mlflow/pull/12137 is merged.